src/documents/activities/InclusiveDesignMapping.md

```diff
@@ -47,25 +57,30 @@ Map your needs, preferences and context on the selected facets. Next, using a di
 Now you can put the facets together, draw a jagged outline, and shade in the gap between the two outlines.
 
 <figure>
-[![A diagram showing facets pieced together into a circle, with mappings](/assets/images/InclusiveDesignMappingTool03.png)](/assets/images/InclusiveDesignMappingTool03.png)
+[![A diagram showing facets pieced together into a circle, with
+mappings](/assets/images/InclusiveDesignMappingTool03.png)](/assets/images/InclusiveDesignMappingTool03.png)
```

Comment:
So same issue as mentioned before, the alt text that is wrapped gets appended to the previous word. I think you'll need to review all of the alt tags. that were touched. There are at least 3 issues with this file alone.